components/director/internal/domain/runtime/service.go

```diff
@@ -302,6 +300,12 @@ func (s *service) CreateWithMandatoryLabels(ctx context.Context, in model.Runtim
 		return errors.Wrapf(err, "while creating multiple labels for Runtime")
 	}
 
+	if scenariosToAssign != nil {
```

Comment:
It was like that before, but the problem is that when assigning the scenario from the input label, the runtimeType label isn't set yet and is empty therefore causing the assigning formation calls to fail because of the newly introduced check.

Reply:
My bad, I overlooked the label setting. We should leave it as is.